### PR TITLE
PB-6468: Change code to take VM backups if kubevirt in `BasicBackupCreation`

### PR DESCRIPTION
**Pull request by kshithijiyer-px**
**What this PR does / why we need it**:

- [x] Change code to take VM backups if kubevirt is installed in `BasicBackupCreation`

Dashboard URL:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/5000/console (With Kubevirt)
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/4991/console (Without Kubevirt)

**Which issue(s) this PR fixes** (optional)
Closes #PB-6468

**Special notes for your reviewer**:

### CODE REVIEW

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Review by sn-px — APPROVED** [review]
LGTM

**Review by ak-px — APPROVED** [review]
LGTM

File: tests/backup/backup_sanity_test.go [hunk 1]
```diff
@@ -244,6 +266,12 @@ var _ = Describe("{BasicBackupCreation}", Label(TestCaseLabelsMap[BasicBackupCre
 				err := CreateBackupWithValidation(ctx, backupName, SourceClusterName, backupLocationName, backupLocationUID, scheduledAppContexts[i:i+1], labelSelectors, BackupOrgID, sourceClusterUid, "", "", "", "")
 				dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of backup [%s]", backupName))
 				backupNames = append(backupNames, backupName)
+				if len(allVirtualMachines) > 0 {
+					backupName := fmt.Sprintf("%s-%s-%v", "autogenerated-backup", scheduledNamespace, time.Now().Unix())
+					err := CreateVMBackup(backupName, allVirtualMachines, SourceClusterName, backupLocationName, backupLocationUID, labelSelectors, BackupOrgID, sourceClusterUid, "", "", "", "", false, ctx)
```
Comment (kphalgun-px) [inline tests/backup/backup_sanity_test.go:271]: How about CreateVMBackupWithValidation?

File: tests/common.go [hunk 2]
```diff
@@ -11105,6 +11107,25 @@ func GetVolumesOnNode(nodeId string) ([]string, error) {
...
+func IsKubevirtInstalled() bool {
+	k8sApiExtensions := apiextensions.Instance()
+	crdList, err := k8sApiExtensions.ListCRDs()
+	if err != nil {
+		return false
+	}
+	for _, crd := range crdList.Items {
+		if crd.Name == "kubevirts.kubevirt.io" {
+			k8sKubevirt := kubevirt.Instance()
+			version, err := k8sKubevirt.GetVersion()
```
Comment (mkoppal-px) [inline tests/common.go:11120]: Better to print the version as well

**Review by mkoppal-px — APPROVED** [review]
LGTM

File: tests/backup/backup_sanity_test.go (hunk 1 above)
Comment (kshithijiyer-px) [reply tests/backup/backup_sanity_test.go]: Done

File: tests/common.go (hunk 2 above)
Comment (kshithijiyer-px) [reply tests/common.go:11120]: Done

**Review by kphalgun-px — APPROVED** [review]
LGTM

**Comment by kshithijiyer-px** [comment]
Successful run https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/5004/console

**Pull request merged**